app/admin/events.rb

```diff
@@ -5,7 +5,7 @@
   #
   # Uncomment all parameters which should be permitted for assignment
   #
-  # permit_params :source, :external_id
+  permit_params :source, :external_id
```

Comment:
lets squash this commit with cb3e566b8df8df287ab1890bf8855749a46c4391 and remove all of these extra comments